assets/price-per-item.js

```diff
@@ -0,0 +1,96 @@
+class PricePerItem extends HTMLElement {
+  constructor() {
+    super();
+    this.input = this.querySelector('input');
+    this.variantId = this.querySelector('input[name="quantity"]').dataset.variantId;
+    this.input.addEventListener('change', this.onInputChange.bind(this));
```

Comment:
We don't have to do this in this PR, but it would be better to add your event listener to an pub/sub. The quantity input publishes an event when its input value changes and the `price-per-item` can subscribe to that event.